src/yachtzee/index.ts

```diff
@@ -1,163 +1,116 @@
-        have($skill`The Polka of Plenty`)
-      ) {
-        useSkill($skill`The Polka of Plenty`);
-      }
-    }
-    garboAdventure($location`The Sunken Party Yacht`, Macro.abort());
-    if (get("lastEncounter") === "Yachtzee!") eventLog.yachtzees += 1;
-    if (myTurncount() > turncount || haveEffect($effect`Fishy`) < fishyTurns) {
-      fishyTurns -= 1;
-      jellyTurns -= 1;
-      turncount = myTurncount();
-      set("_stenchJellyChargeTarget", get("_stenchJellyChargeTarget", 0) - 1);
-    }
-    if (plantCrookweed && FloristFriar.have() && FloristFriar.Crookweed.available()) {
-      FloristFriar.Crookweed.plant();
-    }
-    plantCrookweed = false;
-    postCombatActions();
+          ...meatOutfit,
+        };
+      },
+      post: () => {
+        if (
+          canAdventure($location`The Spooky Forest`) &&
+          FloristFriar.have() &&
+          FloristFriar.Crookweed.available()
+        ) {
+          FloristFriar.Crookweed.plant();
+        }
+      },
+    },
+    {
+      name: "Fishy Pipe",
+      ready: () => have($item`fishy pipe`) && !have($effect`Fishy`),
+      completed: () => get("_fishyPipeUsed"),
+      do: () => use($item`fishy pipe`),
+    },
+    {
+      name: "Spikolodon Spikes",
+      ready: () => have($item`Jurassic Parka`) && get("_spikolodonSpikeUses") < 5,
+      completed: () => get("noncombatForcerActive"),
+      do: () => {
+        const run = tryFindFreeRun(freeRunConstraints(false)) ?? ltbRun();
+        const familiar =
+          run.constraints.familiar?.() ?? freeFightFamiliar({ allowAttackFamiliars: false });
+        run.constraints.preparation?.();
+        freeFightOutfit({ shirt: $item`Jurassic Parka`, ...toSpec(run), familiar }).dress();
+        cliExecute("parka spikolodon");
 
-    doSausage();
-  }
-}
+        const targetZone = $location`Sloppy Seconds Diner`;
+        const macro = Macro.familiarActions()
+          .skill($skill`Launch spikolodon spikes`)
+          .step(run.macro);
 
-export function oldyachtzeeChain(): void {
-  if (!globalOptions.prefs.yachtzeechain) return;
-  if (get("_garboYachtzeeChainCompleted", false)) return;
-  print("Running Yachtzee Chain", "purple");
-  _yachtzeeChain();
-  set("_garboYachtzeeChainCompleted", true);
-  globalOptions.prefs.yachtzeechain = false;
-  if (!globalOptions.nodiet) {
-    shrugIrrelevantSongs();
-    runDiet();
-    prepRobortender(); // Recompute robo drinks' worth after diet is finally consumed
-  }
-  freeRunFights();
-  postFreeFightDailySetup();
+        const ncSkipper = wanderer().unsupportedChoices.get(targetZone);
+        if (ncSkipper) propertyManager.setChoices(ncSkipper);
+        garboAdventureAuto(targetZone, macro);
+        postCombatActions();
+      },
+    },
+    {
+      name: "Clara's Bell",
+      ready: () => have($item`Clara's bell`) && !get("_claraBellUsed"),
+      completed: () => get("noncombatForcerActive"),
+      do: () => use($item`Clara's bell`),
+    },
+  ];
```

Comment:
We seem to both be saving cinch-forces when SBB is open _and_ not using them here. We should steamline that logic--I think not using cinch makes sense (I'm not sure if it's right, but it makes sense), but not the way things are currently set up